File: check_db_clients.py
import psycopg2

try:
    conn = psycopg2.connect("postgresql://postgres:kadrilex-secure-db-2026@37.59.99.86:5432/postgres")
    cur = conn.cursor()
    
    cur.execute('SELECT "numeroClient", "raisonSociale" FROM "Client" ORDER BY "numeroClient" ASC LIMIT 5;')
    print("Top 5 by numeroClient:")
    for row in cur.fetchall():
        print(row)
        
    cur.execute('SELECT "numeroClient", "raisonSociale" FROM "Client" WHERE "raisonSociale" ILIKE \'%Star%\';')
    print("\nSté Star:")
    for row in cur.fetchall():
        print(row)
        
    cur.close()
    conn.close()
except Exception as e:
    print(f"Error: {e}")

File: check_db_clients.ts
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()

async function main() {
    const clients = await prisma.client.findMany({
        orderBy: { numeroClient: 'asc' },
        take: 10,
        select: {
            id: true,
            numeroClient: true,
            raisonSociale: true,
            nom: true,
            prenom: true
        }
    })
    console.log("Top 10 clients sorted by numeroClient:")
    console.log(JSON.stringify(clients, null, 2))
    
    const steStar = await prisma.client.findFirst({
        where: { raisonSociale: { contains: 'Sté Star', mode: 'insensitive' } },
        select: { numeroClient: true, raisonSociale: true }
    })
    console.log("\nSté Star:")
    console.log(steStar)
}

main().catch(console.error).finally(() => prisma.$disconnect())
